manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2112.20
+PackageLocale: en-US
+Publisher: gdiObjects
+PublisherUrl: https://www.gdiobjects.com/index.php
+PublisherSupportUrl: https://www.gdiobjects.com/helpdesk/
+PrivacyUrl: https://www.gdiobjects.com/privacy.php
+Author: gdiObjects
+PackageName: xBar
+PackageUrl: https://www.gdiobjects.com/Products/xBar/Index.php
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: A Dockable Infobar and App Launcher for Windows
+Description: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. Quickly Organize Shortcuts into User-defined Categories. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
+Moniker: xBar

```

```suggestion
Moniker: xbar
```
This is meant to be in lower-case.